src/main/resources/resources/logisim/strings/std/std_de.properties

```diff
@@ -768,6 +768,7 @@ TTL7411 = 7411: Dreifach 3-Eingangs-UND-Gatter
 TTL74125 = 74125: Vierfach Bus-Puffer, Drei-Status-Ausgänge, negative Freigabe
 TTL74157 = 74157: Vierfach 2 zu 1 Leitungsauswahl
 TTL74158 = 74158: Vierfach 2 zu 1 Leitungsauswahl, negierter Ausgang
+TTL74138 = 74138: 3 zu 8 Decoder
```

Comment:
isn't "line" thing important here?

Reply:
I would think so, but I was following the lead of all the surrounding translations that don't use it. (157,158,139).

I should probably just leave it empty, right?